lib/src/Pos/Binary/Slog.hs

```diff
@@ -1,6 +1,6 @@
 -- | Binary serialization of Slog types.
 
-module Pos.Binary.Block.Slog
+module Pos.Binary.Slog
```

Comment:
Why?

Reply:
Well, maybe we should separate `Block` and `Slog`, but currently `Slog` is part of `Block`, so I don't see a reason why this module isn't called `Pos.Binary.Block.Slog` 🤷‍♂️